cfme/containers/overview.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from functools import partial
+from utils.appliance import Navigatable
+from cfme.common import Taggable
+from utils.appliance.implementations.ui import CFMENavigateStep, navigator
+from navmazing import NavigateToAttribute
+from cfme.web_ui import match_location
+
+
+match_page = partial(match_location, controller='container_dashboard', title='Container Dashboards')
```

Comment:
@psav at first glance the use of match-page looks like we have a implicit dependency on browser, do we want to resolve that (i believe this is a general problem not inhibiting a merge of this pr)